src/v2/view-builder/ViewFactory.js

```diff
@@ -97,6 +103,18 @@ const VIEWS_MAPPING = {
     'security_question': EnrollAuthenticatorSecurityQuestion,
     email: EnrollAuthenticatorEmailView
   },
+  [RemediationForms.ENROLL_POLL]: {
+    app: EnrollAuthenticatorOVPollView,
+  },
+  [RemediationForms.ENROLL_SELECT_CHANNEL]: {
+    app: EnrollOVSwitchChannelView,
+  },
+  [RemediationForms.ENROLL_WITH_EMAIL]: {
```

Comment:
Just as follow up:
what do you think to have a generic form like `enroll-via-channel` leave authenticator type / channel to the context data?

Reply:
I feel `ENROLL_WITH_EMAIL` is a bit confusing with `Enroll Authenticator - Email` when people just look at name and likely to ask same question

Reply:
that makes sense. ENROLL_AUTHENTICATOR_WITH_EMAIL